src/authentication/create-email-account/container.test.tsx

```diff
@@ -80,13 +80,13 @@ describe('Container', () => {
       it('weak password', () => {
         const props = subject({ errors: [AccountCreationErrors.PASSWORD_TOO_WEAK] });
 
-        expect(props.errors.password[0]).toEqual('Password must:');
+        expect(props.errors.password).toEqual('Must include at least 8 characters, 1 number and 1 uppercase letter');
       });
 
       it('invalid password', () => {
         const props = subject({ errors: [AccountCreationErrors.PASSWORD_INVALID] });
 
-        expect(props.errors.password[0]).toEqual('Password must:');
+        expect(props.errors.password).toEqual('Must include at least 8 characters, 1 number and 1 uppercase letter');
```

Comment:
haha yes it does reject it
<img width="377" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/e87584e9-a4fe-40f9-a9db-66b8fa5f988f">

Maybe talk to Eth then, tomorrow in the design session - we would need to re-add the `lowercase` condition?

Reply:
and yep, if i use `RATiK123` instead of `RATIK123` then it works
<img width="398" alt="image" src="https://github.com/zer0-os/zOS/assets/33264364/c2867a3a-9b84-4813-ac66-7e835983e8ce">

Reply:
I think you can ask him on your story or just message him. Our meeting tomorrow will be focused on other things.